__tests__/app.test.js

```diff
@@ -397,12 +397,138 @@ describe('9. GET /api/users', ()=>{
             });
         });
     })
-    test('404, returns Not Found when invalid API is given', ()=>{
+    test('404: returns Not Found when invalid API is given', ()=>{
         return request(app)
         .get('/api/CreateTable')
         .expect(404)
         .then(({body: {msg}})=>{
             expect(msg).toBe('Not Found');
         })
     });
-})
\ No newline at end of file
+})
+
+describe('10. GET /api/articles?queries', ()=>{
```

Comment:
It would be good to test you get an empty array back when passed a valid query (eg. topic=paper) with no articles.